src/Umbraco.Core/Cache/ObjectCacheAppCache.cs

```diff
@@ -34,22 +31,16 @@ public class ObjectCacheAppCache : IAppPolicyCache, IDisposable
     /// Initializes a new instance of the <see cref="ObjectCacheAppCache" />.
     /// </summary>
     public ObjectCacheAppCache()
-        : this(Options.Create(new MemoryCacheOptions()), NullLoggerFactory.Instance, null)
+        : this(new MemoryCacheOptions(), NullLoggerFactory.Instance)
     { }
 
     /// <summary>
     /// Initializes a new instance of the <see cref="ObjectCacheAppCache" /> class.
     /// </summary>
     /// <param name="options">The options.</param>
     /// <param name="loggerFactory">The logger factory.</param>
-    /// <param name="hostEnvironment">The host environment.</param>
-    public ObjectCacheAppCache(IOptions<MemoryCacheOptions> options, ILoggerFactory loggerFactory, IHostEnvironment? hostEnvironment)
-    {
-        _options = options;
-        _hostEnvironment = hostEnvironment;
-
-        MemoryCache = new MemoryCache(_options, loggerFactory);
-    }
+    public ObjectCacheAppCache(IOptions<MemoryCacheOptions> options, ILoggerFactory loggerFactory)
```

Comment:
This constructor is currently not directly used by `CoreScope` and `AppCaches` (or tests), so there's currently no way to customize the options (to set size limits) and/or set the logger factory (to add debug logging). If we don't need this, we could either remove this constructor or make it internal...

Reply:
Would be okay as internal

Reply:
```suggestion
    internal ObjectCacheAppCache(IOptions<MemoryCacheOptions> options, ILoggerFactory loggerFactory)
```